modules/ROOT/pages/release-notes/release-notes55.adoc

```diff
@@ -220,10 +222,10 @@ The {productname} 5.5 release includes an accompanying release of the *Accessibi
 
 *Accessibility Checker* 2.3.0 adds the following features:
 
-* A new `a11ychecker_ignored_rules` option for filtering Accessibility Checker rules using the rule identifiers. For information on the `a11ychecker_ignored_rules` option, see: link:{baseurl}/plugins/premium/a11ychecker/#a11ychecker_ignored_rules[Accessibility Checker plugin - `a11ychecker_ignored_rules`].
-* A new `a11ychecker_issue_url_callback` option to allow custom URLs to be used for the "more info" links on accessibility checks. For information on the `a11ychecker_issue_url_callback` option, see: link:{baseurl}/plugins/premium/a11ychecker/#a11ychecker_issue_url_callback[Accessibility Checker plugin - `a11ychecker_issue_url_callback`].
-* Adds the rule identifier (`id`) to the data returned by the `getReport()` API method. For information on the `getReport()` API method, see: link:{baseurl}/plugins/premium/a11ychecker/#getreport[Accessibility Checker plugin - `getReport()` API method].
-* A new `a11ychecker_filter_issues` option for filtering Accessibility Checker report results. For information on the `a11ychecker_filter_issues` option, see: link:{baseurl}/plugins/premium/a11ychecker/#a11ychecker_filter_issue[Accessibility Checker plugin - `a11ychecker_filter_issues`].
+* A new `a11ychecker_ignored_rules` option for filtering Accessibility Checker rules using the rule identifiers. For information on the `a11ychecker_ignored_rules` option, see: xref:plugins/premium/a11ychecker.adoc#a11ychecker_ignored_rules[Accessibility Checker plugin - `a11ychecker_ignored_rules`].
+* A new `a11ychecker_issue_url_callback` option to allow custom URLs to be used for the "more info" links on accessibility checks. For information on the `a11ychecker_issue_url_callback` option, see: xref:plugins/premium/a11ychecker.adoc#a11ychecker_issue_url_callback[Accessibility Checker plugin - `a11ychecker_issue_url_callback`].
+* Adds the rule identifier (`id`) to the data returned by the `getReport()` API method. For information on the `getReport()` API method, see: xref:plugins/premium/a11ychecker.adoc#getreport[Accessibility Checker plugin - `getReport()` API method].
```

Comment:
This xref doesn't resolve as the `getreport` id does exist on the heading in `a11ychecker.adoc`. e.g:
```
[[getreport]]
=== `getReport()`
```

Reply:
done